src/main/java/pro/sky/skyprospring/controller/NewDepartmentController.java

```diff
@@ -0,0 +1,48 @@
+package pro.sky.skyprospring.controller;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import pro.sky.skyprospring.Service.impl.DepartmentServiceImpl;
+import pro.sky.skyprospring.model.Employee;
+
+import java.util.List;
+import java.util.Map;
+
+@RestController
+@RequestMapping("/department")
+public class NewDepartmentController {
+    private final DepartmentServiceImpl departmentService;
+
+    public NewDepartmentController(DepartmentServiceImpl departmentService) {
```

Comment:
```suggestion
    public NewDepartmentController(DepartmentService departmentService) {
```
инъекцию надо делать по интерфейсу